examples/torch/object_detection/dataset.py

```diff
@@ -138,11 +137,21 @@ def base_transform(image, size, mean, std, normalize_coef):
 
 
 class BaseTransform:
-    def __init__(self, size, mean, std, normalize_coef):
+    def __init__(self,
+                 size: int,
+                 mean: int,
```

Comment:
I doubt that mean, std and normalize_coef values are expected to be integer only.

Reply:
done